t/139-ssl-cert-by.t

```diff
@@ -2316,3 +2316,206 @@ ssl handshake: userdata
 uthread: hello from f()
 uthread: killed
 uthread: failed to kill: already waited or killed
+
+
+
+=== TEST 27: get server_port - IPv4
```

Comment:
Are you duplicating the tests between lua-nginx-module and lua-resty-core? Please don't do that.

Reply:
moved all tests to lua-resty-core